oarepo_runtime/ext.py

```diff
@@ -226,3 +238,107 @@ def draft_indices(self) -> set[str]:
                 if draft_index is not None:
                     indices.add(draft_index.search_alias)
         return indices
+
+    @overload
+    def get_export_from_record_item(
+        self,
+        record_item: dict,
+        representation: Literal[ExportRepresentation.RESPONSE],
+        export_code: str | None = None,
+        export_mimetype: str | None = None,
+    ) -> tuple[Any, int, dict[str, str]] | None: ...
+
+    @overload
+    def get_export_from_record_item(
+        self,
+        record_item: dict,
+        representation: Literal[ExportRepresentation.DICTIONARY],
+        export_code: str | None = None,
+        export_mimetype: str | None = None,
+    ) -> dict: ...
+
+    @overload
+    def get_export_from_record_item(
```

Comment:
now, when the record_item is in fact a serialized dict (record_item.to_dict is called outside this code), should not we rename the variable and function - for example serialized_record and get_export_from_serialized_record?

please make the docstring for the (current) record_item up to date.